sdk/state/integration/state_test.go

```diff
@@ -454,10 +452,19 @@ func TestOpenUpdatesCoordinatedClose(t *testing.T) {
 	accountRequest := horizonclient.AccountRequest{AccountID: responder.Escrow.Address()}
 	responderEscrowResponse, err := client.AccountDetail(accountRequest)
 	require.NoError(t, err)
-	assert.EqualValues(t, fmt.Sprintf("%.7f", float64(rBalanceCheck)/float64(1_000_0000)), responderEscrowResponse.Balances[0].Balance)
+	assert.EqualValues(t, fmt.Sprintf("%.7f", float64(rBalanceCheck)/float64(1_000_0000)), assetBalance(asset, responderEscrowResponse))
 
 	accountRequest = horizonclient.AccountRequest{AccountID: initiator.Escrow.Address()}
 	initiatorEscrowResponse, err := client.AccountDetail(accountRequest)
 	require.NoError(t, err)
-	assert.EqualValues(t, fmt.Sprintf("%.7f", float64(iBalanceCheck)/float64(1_000_0000)), initiatorEscrowResponse.Balances[0].Balance)
+	assert.EqualValues(t, fmt.Sprintf("%.7f", float64(iBalanceCheck)/float64(1_000_0000)), assetBalance(asset, initiatorEscrowResponse))
+}
+
+func assetBalance(asset txnbuild.Asset, account horizon.Account) string {
```

Comment:
helper function that gets the balance for a specific asset from a `horizon.Account` response

Reply:
super intuitive to read this function used in the code above 💯 🎉